examples/ieee802154/ieee802154_receive_all_frames/.cargo/config.toml

```diff
@@ -5,8 +5,20 @@ rustflags = [
   "-C", "force-frame-pointers",
 ]
 
+[target.'cfg(target_arch = "xtensa")']
```

Comment:
Curious why you decided to add Xtensa configuration for the 802.15.4 examples? None of the Xtensa chips have this radio, and given we don't plan on releasing any new Xtensa chips this will never be used.